python/ray/dashboard/client/src/pages/dashboard/Tune.tsx

```diff
@@ -0,0 +1,125 @@
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import createStyles from "@material-ui/core/styles/createStyles";
+import withStyles, { WithStyles } from "@material-ui/core/styles/withStyles";
+import React from "react";
+import { connect } from "react-redux";
+import { getTuneInfo } from "../../api";
+import { StoreState } from "../../store";
+import { dashboardActions } from "./state";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      padding: theme.spacing(2),
+      "& > :not(:first-child)": {
+        marginTop: theme.spacing(2)
+      }
+    },
+    table: {
+      marginTop: theme.spacing(1)
+    },
+    cell: {
+      padding: theme.spacing(1),
+      textAlign: "center",
+      "&:last-child": {
+        paddingRight: theme.spacing(1)
+      }
+    }
+  });
+
+const mapStateToProps = (state: StoreState) => ({
+  tuneInfo: state.dashboard.tuneInfo,
+});
+
+const mapDispatchToProps = dashboardActions;
+
+class Tune extends React.Component<WithStyles<typeof styles> &
+  ReturnType<typeof mapStateToProps> &
+  typeof mapDispatchToProps> {
+  interval: number = 0;
+
+  refreshTuneInfo = async () => {
+    try {
+      const [tuneInfo] = await Promise.all([
+        getTuneInfo()
+      ]);
+      this.props.setTuneInfo({ tuneInfo });
+    } catch (error) {
+      this.props.setError(error.toString());
+    }
+  };
+
+  async componentDidMount() {
+    this.interval = window.setInterval(this.refreshTuneInfo, 1000);
+  }
+
+  async componentWillUnmount() {
+    window.clearInterval(this.interval);
+  }
+
+  render() {
+    const { classes, tuneInfo } = this.props;
+
+    if (tuneInfo === null || Object.keys(tuneInfo["trial_records"]).length === 0) {
+      return null;
+    }
+
+    const first_trial = Object.keys(tuneInfo["trial_records"])[0];
+    let paramNames: string[] = [];
+    if (tuneInfo !== null) {
+      let paramsDict = tuneInfo["trial_records"][first_trial]["params"];
+      paramNames = Object.keys(paramsDict).filter((k) => (k !== "args"));
+    }
+
+    const metricNames = Object.keys(tuneInfo["trial_records"][first_trial]["metrics"]);
+
+    return (
+      <div className={classes.root}>
+        <Table className={classes.table}>
+          <TableHead>
+            <TableRow>
+              <TableCell className={classes.cell}>Trial ID</TableCell>
+              <TableCell className={classes.cell}>Job ID</TableCell>
+              <TableCell className={classes.cell}>Start Time</TableCell>
+              {paramNames.map((value, index) => (
+                <TableCell className={classes.cell}>{value}</TableCell>
```

Comment:
Make sure to run the dashboard using the React development server and `RAY_DASHBOARD_DEV=1 python ...`, and take a look at the console to view any warnings. For instance, there is a probably a warning here about missing `key` attributes for elements in an array.